docs/readthedocs/models/chp_model.md

```diff
@@ -0,0 +1,58 @@
+(chp_model)=
+
+## Chp Model
+
+This page documents the functionality of the CHP Model (combined heat and power plant model) available in SIMONA. This model is part of the SIMONA simulation framework and represented by an agent. The current model is feasible to simulate nano- and micro-CHP units. As it is a cross-sector agent heat/electricity) the usage of a house-heat-model is mandatory.
+
+### Attributes, Units and Remarks
+
+
+| Attribute         | Unit | Remarks                                         |
+| ------------------- | ------ | ------------------------------------------------- |
+| uuid              | --   | universally unique identifier                   |
+| id                | --   | human readable identifier                       |
+| operationInterval | --   | time interval, in which the system is operating |
+| scalingFactor     | --   | scaling the output of the system                |
+| qControl          | --   | type of reactive power control                  |
+| sRated            | kVA  | rated apparent power                            |
+| cosPhiRated       | --   | rated power factor                              |
+| pThermal          | kW   | rated thermal power (generated by CHP unit)     |
+| storage           | --   | thermal storage                                 |
```

Comment:
Does it make sense to replicate the input parameters from PowerSystemDataModel? Moreover, would it make sense to link to the input models?

Reply:
Like it! :smiley: